index.html

```diff
@@ -1224,6 +1204,46 @@ <h3>Issuance Date</h3>
         </pre>
       </section>
 
+      <section>
+        <h3>Credential Subject</h3>
+
+        <p>
+A <a>verifiable credential</a> contains <a>claims</a> about a <a>subject</a>.
+The <a>subject</a> of the credential MUST be expressed using the
+<code>credentialSubject</code> <a>property</a>.
+        </p>
+
+        <dl>
+          <dt>credentialSubject</dt>
+          <dd>
+The value of the <code>credentialSubject</code> <a>property</a> MUST be
+an object that contains one or more properties that are related to the
+<a>subject</a> of the <a>verifiable credential</a>. The object MAY contain an
```

Comment:
related to this subject of the

Reply:
Because there can be more than one credential subject

Reply:
Fixed.